### ISSUE TITLE
Syntax highlighting requires blank between ⋄ and :endif

### ISSUE THREAD

**Issue by mbaas2** [OptimaSystems/vscode-apl-language#1]
Does not color `...⋄:endif`correctly.

**Comment by aplteam** [comment]
Well, putting `⋄` and control structures on the same line is a **very** bad idea. So, rather putting an effort into "fixing" this my advice is: don't do this.

As long as the Dyalog debugger does not process diamondized lines properly, putting them on one line defeats the whole purpose of control structures: you won't see which code is executed and which isn't.

**Comment by mbaas2** [comment]
I agree that a better handling in the tracer would be nice. But I disagree with the conclusion about not fixing this. 
NB: This would also affect cases like
```
:if Condition
   DoThis ⋄:endif
```
which are frequently use. That can be traced/debugged just fine, but the syntax colouring does not kick in for the 2nd line.
